src/jupytext/parse_version.py

```diff
@@ -1,11 +1,10 @@
 def parse_version(version, custom_error=ImportError):
     try:
-        from pkg_resources import parse_version as parse
+        from packaging.version import parse
```

Comment:
I think it is worth dropping `pkg_resources` and make `packaging` a dependency. The former is not readily installable on pypi anymore.

Reply:
Done

Reply:
Can also just remove `parse_version.py` file

Reply:
Oh yes sure (that was the original intent obviously)